Library/Formula/goolabs.rb

```diff
@@ -1,8 +1,8 @@
 class Goolabs < Formula
   desc "Command-line tool for morphologically analyzing Japanese language"
   homepage "https://pypi.python.org/pypi/goolabs"
-  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.1.1.tar.gz"
-  sha1 "3201102ab0fad77d57a09dd7c879f6f163e05799"
+  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.0.tar.gz"
+  sha1 "c544460b51ad63c7d79b0ef542816c2981fa22f9"
```

Comment:
@mikemcquaid 
I have just fixed it. and upgrade goolabs new minor version 0.2.2.
Please feel free to check again :)

thanks!